pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an [event][] occurs when, by comparing the [web page][] before and 1 minute after the event [firing][], at least one of the following occurs:
```

Comment:
I'm considering
```suggestion
A _change in the content of a [web page][] caused by an [event][]_ occurs when, by comparing the [web page][] before and 1 minute after the event [firing][], at least one of the following occurs:
```
to emphasise that the event causing it is an important part of the definition (vs, say, time-based media).
Not super happy about this because the "of a web page" in the middle is diluting the intention of emphasising "change in the content caused by an event"…

Reply:
I think I got a way to address that